src/StructuredLogViewer.Avalonia/App.xaml

```diff
@@ -219,8 +219,8 @@
                           ItemsSource="{Binding Children}">
             <StackPanel Orientation="Horizontal">
                 <ProgressBar Width="40" 
-                             Height="9" 
-                             Margin="2,1,6,0" 
+                             Height="12" 

```

Comment:
We can revert the "Fix other width/height, margin/padding properties" commit. It is optional, it looks like

Reply:
Feel free to do what you feel is right